crates/consensus/src/malachite/address.rs

```diff
@@ -1,34 +1,48 @@
 use pathfinder_crypto::Felt;
 use serde::{Deserialize, Serialize};
 
-/// A validator address for the malachite context.
-///
-/// This is a wrapper around the `ContractAddress` type from the
-/// `pathfinder_common` crate which implements the `Address` trait for the
-/// malachite context.
+/// A validator address used to identify participants in the consensus protocol.
 #[derive(Ord, PartialOrd, Eq, PartialEq, Clone, Copy, Default, Hash, Serialize, Deserialize)]
 pub struct ValidatorAddress(pathfinder_common::ContractAddress);
 
 impl ValidatorAddress {
+    /// Create a new validator address from a contract address.
+    pub fn new(address: pathfinder_common::ContractAddress) -> Self {
+        Self(address)
+    }
+
+    /// Create a validator address from a felt value.
+    pub fn from_felt(felt: Felt) -> Self {
+        Self(pathfinder_common::ContractAddress::new(felt).unwrap_or_default())
```

Comment:
I'm not sure defaulting to ZERO is a sensible thing to do here. Shouldn't this constructor be fallible instead?